contrib/cmdclient/console.py

```diff
@@ -718,7 +718,7 @@ def _run_and_pprint(
         method,
         path,
         data=None,
-        query_params={"access_token": None},
+        query_params: dict = None,
```

Comment:
as discussed on synapse-dev: I'd prefer it if these were explictly annotated with Optional rather than relying on mypy's `implicit-optional`.